sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/ConversationAnalysisClient.cs

```diff
@@ -150,5 +153,136 @@ public virtual Response<AnalyzeConversationTaskResult> AnalyzeConversation(strin
                 throw;
             }
         }
+
+        /// <summary>Analyzes a conversational utterance.</summary>
+        /// <param name="input">The <see cref="GeneratedConversation"/> used for tasks input.</param>
+        /// <param name="tasks"> <see cref="AnalyzeConversationLROTask"/> defines the tasks to be used.</param>
+        /// <param name="cancellationToken">An optional <see cref="CancellationToken"/> to cancel the request.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="input"/> or <paramref name="tasks"/> or is null.</exception>
+        /// <exception cref="RequestFailedException">The service returned an error. The exception contains details of the service error.</exception>
+        public virtual async Task<Response<AnalyzeConversationJobState>> AnalyzeConversationAsync(IEnumerable<GeneratedConversation> input, IEnumerable<AnalyzeConversationLROTask> tasks, CancellationToken cancellationToken = default)
+        {
+            Argument.AssertNotNullOrEmpty(input, nameof(input));
+            Argument.AssertNotNull(tasks, nameof(tasks));
+
+            MultiLanguageConversationAnalysisInput multiLanguageConversationAnalysisInput = new MultiLanguageConversationAnalysisInput(input);
+
+            var analyzeConversationJobsInput = new AnalyzeConversationJobsInput(multiLanguageConversationAnalysisInput, tasks);
+
+            using DiagnosticScope scope = Diagnostics.CreateScope($"{nameof(ConversationAnalysisClient)}.{nameof(AnalyzeConversation)}");
+            scope.Start();
+
+            try
+            {
+                var responseHeaders = await _analysisRestClient.SubmitJobAsync(analyzeConversationJobsInput, cancellationToken).ConfigureAwait(false);
+                var jobId = getJobId(responseHeaders.Headers.OperationLocation);
+
+                await WaitForJobUntilDoneAsync(jobId).ConfigureAwait(false);
```

Comment:
We should be returning an `Operation<T>` and callers wait.

Reply:
In fact, the generated client should already be doing this automatically. Just call it with whatever parameters you need to constructed here in these pair of methods.